src/quo2/components/settings/reorder_item/items/item.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.settings.reorder-item.items.item
+  (:require [react-native.core :as rn]
+            [quo2.components.settings.reorder-item.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.icon :as icon]
+            [quo2.foundations.colors :as colors]))
+
+(defn view
+  [{:keys
+    [title
+     subtitle
+     image
+     image-size
+     right-text
+     right-icon
+     on-press]}]
+  [rn/touchable-opacity
+   {:on-press on-press
+    :style    (merge (style/item-container) (when subtitle style/item-container-extended))}
+   [icon/icon :main-icons/drag
+    {:color  (colors/theme-colors
+              colors/neutral-50
+              colors/neutral-40)
+     :width  19
```

Comment:
width and height is redundant, also drag.png is redundant , we already have it in icons2 folder which is used by icon component, could you pls remove pngs

Reply:
Done